beacon_node/http_metrics/src/metrics.rs

```diff
@@ -41,6 +42,8 @@ pub fn gather_prometheus_metrics<T: BeaconChainTypes>(
 
     warp_utils::metrics::scrape_health_metrics();
 
+    scrape_allocator_metrics();
```

Comment:
In case we keep the potentially unsafe `mallinfo`, perhaps we should guard this call so that it only runs when `--disable-malloc-tuning` isn't set.

Reply:
Good call!